File: eBotanika.Services/Database/Rezervacije.cs
namespace eBotanika.Services.Database
{
    public partial class Rezervacije
    {
        public Rezervacije() {
            Dostava = new HashSet<Dostava>();
        }

        public int RezervacijaID { get; set; }
        public DateTime DatumRezervacije { get; set; }
        public int KorisnikID { get; set; }
        public Korisnik Korisnik { get; set; }
        public int GradID { get; set; }
        public Gradovi Grad { get; set; }
        public string Napomena { get; set; }
        public int Kolicina { get; set; }
        public int SvrhaID { get; set; }
        public string AdresaDostave { get; set; }
        public Svrha Svrha { get; set; }
        public int BiljkeID { get; set; }
        public Biljke Biljke { get; set; }
        public int OcjenaID { get; set; }
        public Ocjena Ocjena { get; set; } 
        public decimal OcjenaUsluge { get; set; }
        public ICollection<Dostava> Dostava { get; set; }
    }
}

File: ebotanika_mobile/lib/models/rezervacije.dart
class Rezervacije {
  final int KorisnikID;
  final int GradID;
  final String DatumRezervacije;
  final String Napomena;
  final String SvrhaID;
  final String AdresaDostave;
  final String Kolicina;
  final int BiljkeID;
  final int OcjenaID;
  final double OcjenaUsluge;

  Rezervacije(
      {
      required this.KorisnikID,
      required this.GradID,
      required this.DatumRezervacije,
      required this.Napomena,
      required this.SvrhaID,
      required this.AdresaDostave,
      required this.Kolicina,
      required this.BiljkeID,
      required this.OcjenaID,
      required this.OcjenaUsluge});

  factory Rezervacije.fromJson(Map<String, dynamic> json) {
    return Rezervacije(
        KorisnikID: json["KorisnikID"],
        GradID: json["GradID"],
        DatumRezervacije: json["DatumRezervacije"],
        Napomena: json["Napomena"],
        SvrhaID: json["SvrhaID"],
        AdresaDostave: json["AdresaDostave"],
        Kolicina: json["Kolicina"],
        BiljkeID: json["BiljkeID"],
        OcjenaID: json["OcjenaID"],
        OcjenaUsluge: json["OcjenaUsluge"]);
  }

  Map<String, dynamic> toJson() => {
        "KorisnikID": KorisnikID,
        "GradID": GradID,
        "DatumRezervacije": DatumRezervacije,
        "Napomena": Napomena,
        "SvrhaID": SvrhaID,
        "AdresaDostave": AdresaDostave,
        "Kolicina": Kolicina,
        "BiljkeID": BiljkeID,
        "OcjenaID": OcjenaID,
        "OcjenaUsluge": OcjenaUsluge
      };
}
